mypy/test/data.py

```diff
@@ -610,12 +622,15 @@ def repr_failure(self, excinfo: Any) -> str:
 
 
 class DataSuite:
+    files = None  # type: typing.ClassVar[List[str]]
+    base_path = '.'  # type: typing.ClassVar[str]
+    optional_out = False  # type: typing.ClassVar[bool]
+    native_sep = False  # type: typing.ClassVar[bool]
+    require_stable = False  # type: typing.ClassVar[bool]
+    require_incremental = False  # type: typing.ClassVar[bool]
```

Comment:
`require_stable` and `require_incremental` are too specialized to be included here as configuration attributes. It's okay to have `is_incremental` and `has_stable_flags` as utility functions in this module (I don't think that they need to be associated with any class), but I'd prefer if these would be explicitly handled using asserts in `run_case` methods, similar to how these were implemented previously.

Reply:
These are also used as filters in the test selection, and the asserts only validate it.  I will add a `filter` method then.